automake/post/rules/makedirs.am

```diff
@@ -35,6 +35,8 @@
 
 force: # no rules, please always build ;)
 
+ifdef MAKEDIRS
```

Comment:
Worth pull this up to before line 34 and re-labeling "force" to "makedirs-force" to avoid namespace pollution or conflicts with other make snippets that might define "force"?

Reply:
yes